nixos/modules/services/monitoring/munin.nix

```diff
@@ -375,7 +372,7 @@ in
 
     # Munin is hardcoded to use DejaVu Mono and the graphs come out wrong if
     # it's not available.
-    fonts.packages = [ pkgs.dejavu_fonts ];
+    fonts.fonts = [ pkgs.dejavu_fonts ];  # fonts.packages in unstable, but fonts.fonts in 2305
```

Comment:
Unrelated change. Please put in a separate commit. (Same PR is fine.)

Reply:
This is what I get for using `git add` without `-p` late at night -- that change shouldn't be merged, it's a local change to let my (nixos-23.03) systems load this (nixos-unstable) module.

I'll revert that, and I also found another munin-module bug related to `internAndFixPlugins` that I want to get a fix for into this branch later this afternoon.